Source/WebCore/platform/LocalizedStrings.cpp

```diff
@@ -553,6 +553,13 @@ String contextMenuItemPDFActualSize()
     return WEB_UI_STRING_WITH_MNEMONIC("Actual Size", "_Actual Size", "Actual Size context menu item");
 }
 
+String contextMenuItemPDFAutoSize()
```

Comment:
Probably better to delete this definition and to update the conditions on the duplicate.

```cpp
#if ENABLE(PDFJS) || ENABLE(UNIFIED_PDF)
String contextMenuItemPDFAutoSize()
{
    return WEB_UI_STRING_WITH_MNEMONIC("Automatically Resize", "_Automatically Resize", "Automatically Resize context menu item");
}
```

Reply:
Or just enable it everywhere with `ENABLE(CONTEXT_MENUS)` and delete the dupe 🤷